fhir-persistence-schema/docs/SchemaToolUsageGuide.md

```diff
@@ -0,0 +1,582 @@
+# IBM FHIR Server - Schema Deployment
+
+This document details how to deploy the IBM FHIR Server schema and apply release upgrades. This document also describes how the schema changes are managed.
+
+The schema tool generates the following object types that require management: 
+
+- DB2 PACKAGE
+- TABLESPACES
+- SCHEMA 
+- GLOBAL VARIABLE 
+- INDEX 
+- PERMISSION 
+- PROCEDURE 
+- SEQUENCE 
+- TABLE
+- TABLE CONSTRAINT 
+
+For details on the schema design, refer to the [Schema Design](https://github.com/IBM/FHIR/tree/master/fhir-persistence-schema/docs/SchemaDesign.md) document.
+
+----------------------------------------------------------------
+## Database Support
+
+
+| Database   | Version   | Support |
+|------------|-----------|-----------------------------------|
+| DB2        |      11.5 | Suitable for production. Supports multi-tenancy. |
+| PostgreSQL |        12 | Single tenant per database. |
```

Comment:
I think the versions listed here are our 'minimum' versions, right?
For example, we can (and do) run with PostgreSQL 13 and 14.  Maybe we can make it more clear?

Reply:
done